foundations/html_css/flexbox/project-landing-page.md

```diff
@@ -60,6 +60,8 @@ If you want to show your work (the project) to others, or submit a solution belo
 
 GitHub allows you to publish web projects directly from a GitHub repository. Doing this will allow you to access your project from `your-github-username.github.io/your-github-repo-name`
 
+Note- Enabling pages on private repo requires paid account.
```

Comment:
Can we use [one of our note boxes](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) for this please?

Reply:
> Can we use [one of our note boxes](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) for this please?

Thanks for informing . i will add the inbuilt note feature.